packages/make-angular-cli-faster/src/utilities/make-angular-cli-faster.ts

```diff
@@ -42,4 +43,27 @@ export async function makeAngularCliFaster(args: Args) {
       'and how it can speed up your CI by up to 10 times at https://nx.dev/getting-started/nx-and-angular',
     ],
   });
+
+  if (migration.incompatibleWithAngularVersion) {
+    output.warn({
+      title:
+        'The provided version of Nx is not compatible with the currently installed version of Angular.',
+      bodyLines: [
+        'The workspace was still migrated, but you might find unexpected issues or behaviors using this version.',
```

Comment:
We should say something along the lines of.. we've migrated you to `Nx v<oldVersion>` for now.

Reply:
Let's figure out what we want to do with https://github.com/nrwl/nx/pull/13591#discussion_r1039338441 first. This might not be needed at all based on that.